gpt_index/output_parsers/selection.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+import json
+from typing import Any
+
+from dataclasses_json import DataClassJsonMixin
+from gpt_index.output_parsers.base import BaseOutputParser, StructuredOutput
+
+
+def escape_curly_braces(input_string):
+    # Replace '{' with '{{' and '}' with '}}' to escape curly braces
+    escaped_string = input_string.replace("{", "{{").replace("}", "}}")
+    return escaped_string
+
+
+FORMAT_STR = """The output should be formatted as a JSON instance that conforms to the JSON schema below. 
+
+
+Here is the output schema:
+{
```

Comment:
should just use pydantic or https://pypi.org/project/dataclasses-jsonschema/ to get JSON schema and populate this properly.

Reply:
did a quick change to this to test the waters, since the old regex logic was flaky